### PR TITLE
fix: Improved print format of Statement of account report

### PR DESCRIPTION
**Pull request by meshramsaroj**
**Task Link:** https://app.asana.com/0/1191634438432931/1191716019962961/f

**Description:** Beautified print format of Statement of account report using CSS style.

**Screenshot:**

- Before

before_SOA

- After

updated-soa

### CODE REVIEW

File: erpnext/accounts/report/statement_of_account/statement_of_account.html
Comment (nehasacher143) [inline erpnext/accounts/report/statement_of_account/statement_of_account.html]: @meshramsaroj Please add a space Bracket and end of style rule
Do it for all other rules

File: erpnext/accounts/report/statement_of_account/statement_of_account.html
Comment (nehasacher143) [inline erpnext/accounts/report/statement_of_account/statement_of_account.html]: Double quotes to the class name

File: erpnext/accounts/report/statement_of_account/statement_of_account.html
Comment (nehasacher143) [inline erpnext/accounts/report/statement_of_account/statement_of_account.html]: Remove an extra space here

File: erpnext/accounts/report/statement_of_account/statement_of_account.html
Comment (nehasacher143) [inline erpnext/accounts/report/statement_of_account/statement_of_account.html]: Why there are spaces before '>' in entire code. Please rectify

File: erpnext/accounts/report/statement_of_account/statement_of_account.html
Comment (nehasacher143) [inline erpnext/accounts/report/statement_of_account/statement_of_account.html]: Can you please explain why this change

**Comment by nehasacher143** [comment]
Hey @Alchez Code looks good to me as well as the design
Please merge this if this is okay to you.

File: erpnext/accounts/report/statement_of_account/statement_of_account.html
Comment (Alchez) [inline erpnext/accounts/report/statement_of_account/statement_of_account.html]: We still need to figure out how to fetch the addresses, I'll have to look at this.

Comment (nehasacher143) [reply erpnext/accounts/report/statement_of_account/statement_of_account.html]: @Alchez Can we not merge it till then?

Comment (Alchez) [reply erpnext/accounts/report/statement_of_account/statement_of_account.html]: No point merging it, really, without the entire format done

**Comment by Alchez** [comment]
@meshramsaroj, resolve the conflicts here

File: erpnext/accounts/report/statement_of_account/statement_of_account.js
Comment (Alchez) [inline erpnext/accounts/report/statement_of_account/statement_of_account.js]:
```suggestion
		return;
```

File: erpnext/accounts/report/statement_of_account/statement_of_account.js
Comment (Alchez) [inline erpnext/accounts/report/statement_of_account/statement_of_account.js]: Talk to @vishdha and Arundhati about this, maybe we don't want to allow multiple parties in that field.

**Review by Alchez — CHANGES_REQUESTED** [review]
@meshramsaroj, the address logic is looking good. You just need to add it to the HTML now.

**Review by Alchez — APPROVED** [review]

**Pull request merged**
